Cannot compile for aarch64

When trying to compile yubihsmrs on a Raspberry Pi 4 with Ubuntu Server, the process fails with the following errors: 

```
error[E0308]: mismatched types
   --> src/object.rs:728:66
    |
728 |             ::error::result_from_libyh(lyh::yh_algo_to_string(a, &mut ptr))
    |                                                                  ^^^^^^^^ expected `u8`, found `i8`
    |
    = note:    expected raw pointer `*mut *const u8`
            found mutable reference `&mut *const i8`

error[E0308]: mismatched types
   --> src/object.rs:733:38
    |
733 |             std::ffi::CStr::from_ptr(ptr)
    |                                      ^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`

error[E0308]: mismatched types
   --> src/object.rs:806:76
    |
806 |             ::error::result_from_libyh(lyh::yh_type_to_string(self.into(), &mut ptr))
    |                                                                            ^^^^^^^^ expected `u8`, found `i8`
    |
    = note:    expected raw pointer `*mut *const u8`
            found mutable reference `&mut *const i8`

error[E0308]: mismatched types
   --> src/object.rs:811:38
    |
811 |             std::ffi::CStr::from_ptr(ptr)
    |                                      ^^^ expected `u8`, found `i8`
    |
    = note: expected raw pointer `*const u8`
               found raw pointer `*const i8`

error: aborting due to 4 previous errors; 23 warnings emitted

For more information about this error, try `rustc --explain E0308`.
error: could not compile `yubihsmrs`

Caused by:
  process didn't exit successfully: `rustc --crate-name yubihsmrs src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 -C embed-bitcode=no -C metadata=9de4ea0e978ebc47 -C extra-filename=-9de4ea0e978ebc47 --out-dir /home/marc/yubihsmrs/target/release/deps -L dependency=/home/marc/yubihsmrs/target/release/deps --extern lazy_static=/home/marc/yubihsmrs/target/release/deps/liblazy_static-f94112527a8f5566.rmeta --extern libyubihsm_sys=/home/marc/yubihsmrs/target/release/deps/liblibyubihsm_sys-f42423c26b931e80.rmeta --extern log=/home/marc/yubihsmrs/target/release/deps/liblog-3f813f59dc7ccf2e.rmeta --extern regex=/home/marc/yubihsmrs/target/release/deps/libregex-70af35c121a3b3fb.rmeta --extern rustc_serialize=/home/marc/yubihsmrs/target/release/deps/librustc_serialize-0020115555e258e8.rmeta -L /usr/local/lib/` (exit status: 1)
```

How can I solve the issue?

--- a/src/object.rs
+++ b/src/object.rs
@@ -720,7 +720,7 @@ impl<'a> From<&'a ObjectAlgorithm> for yh_algorithm {
 
 impl std::fmt::Display for ObjectAlgorithm {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        let mut ptr: *const i8 = std::ptr::null();
+        let mut ptr: *const std::os::raw::c_char = std::ptr::null();
 
         let a: yh_algorithm = self.into();
 
@@ -800,7 +800,7 @@ impl<'a> From<&'a ObjectType> for yh_object_type {
 
 impl std::fmt::Display for ObjectType {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        let mut ptr: *const i8 = std::ptr::null();
+        let mut ptr: *const std::os::raw::c_char = std::ptr::null();
 
         try!(unsafe {
             ::error::result_from_libyh(lyh::yh_type_to_string(self.into(), &mut ptr))

